### ISSUE TITLE
recommended learning rate

### ISSUE THREAD

**Issue by srttt** [litwellchi/videocrafter-training-pytorch#3]
what is the recommended learning rate for finetuning the model

**Comment by litwellchi** [comment]
Hi, thanks for your attention on this repo. The recommended lr is about to be 1.0e-05, while we also noticed that this repo has some un-expected errors during training, so maybe you can refer to our new codebase VideoTuna for more convenient training or inference
https://github.com/VideoVerses/VideoTuna
